[2025-10-03T16:17:18.526+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-03T16:17:18.575+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dbt_dag.dbt_tests scheduled__2025-10-02T00:00:00+00:00 [queued]>
[2025-10-03T16:17:18.593+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dbt_dag.dbt_tests scheduled__2025-10-02T00:00:00+00:00 [queued]>
[2025-10-03T16:17:18.594+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-10-03T16:17:18.623+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): dbt_tests> on 2025-10-02 00:00:00+00:00
[2025-10-03T16:17:18.636+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=672) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-03T16:17:18.639+0000] {standard_task_runner.py:64} INFO - Started process 674 to run task
[2025-10-03T16:17:18.641+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dbt_dag', 'dbt_tests', 'scheduled__2025-10-02T00:00:00+00:00', '--job-id', '65', '--raw', '--subdir', 'DAGS_FOLDER/dbt_dag.py', '--cfg-path', '/tmp/tmpqtw3tbx8']
[2025-10-03T16:17:18.646+0000] {standard_task_runner.py:91} INFO - Job 65: Subtask dbt_tests
[2025-10-03T16:17:18.744+0000] {task_command.py:426} INFO - Running <TaskInstance: dbt_dag.dbt_tests scheduled__2025-10-02T00:00:00+00:00 [running]> on host 88331542f089
[2025-10-03T16:17:18.926+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dbt_dag' AIRFLOW_CTX_TASK_ID='dbt_tests' AIRFLOW_CTX_EXECUTION_DATE='2025-10-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-02T00:00:00+00:00'
[2025-10-03T16:17:18.930+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-03T16:17:18.932+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-10-03T16:17:18.934+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /opt/***/dbt && DBT_PROFILES_DIR=/opt/***/dbt dbt test']
[2025-10-03T16:17:18.958+0000] {subprocess.py:86} INFO - Output:
[2025-10-03T16:17:23.078+0000] {subprocess.py:93} INFO - [0m16:17:23  Running with dbt=1.10.13
[2025-10-03T16:17:24.023+0000] {subprocess.py:93} INFO - [0m16:17:24  Registered adapter: postgres=1.9.1
[2025-10-03T16:17:24.717+0000] {subprocess.py:93} INFO - [0m16:17:24  Unable to do partial parsing because config vars, config profile, or config target have changed
[2025-10-03T16:17:28.422+0000] {subprocess.py:93} INFO - [0m16:17:28  Found 4 models, 2 data tests, 1 source, 447 macros
[2025-10-03T16:17:28.432+0000] {subprocess.py:93} INFO - [0m16:17:28
[2025-10-03T16:17:28.435+0000] {subprocess.py:93} INFO - [0m16:17:28  Concurrency: 1 threads (target='dev')
[2025-10-03T16:17:28.438+0000] {subprocess.py:93} INFO - [0m16:17:28
[2025-10-03T16:17:28.949+0000] {subprocess.py:93} INFO - [0m16:17:28  1 of 2 START test not_null_stg_appointments_appointment_id ..................... [RUN]
[2025-10-03T16:17:29.302+0000] {subprocess.py:93} INFO - [0m16:17:29  1 of 2 PASS not_null_stg_appointments_appointment_id ........................... [[32mPASS[0m in 0.35s]
[2025-10-03T16:17:29.310+0000] {subprocess.py:93} INFO - [0m16:17:29  2 of 2 START test unique_stg_appointments_appointment_id ....................... [RUN]
[2025-10-03T16:17:29.729+0000] {subprocess.py:93} INFO - [0m16:17:29  2 of 2 PASS unique_stg_appointments_appointment_id ............................. [[32mPASS[0m in 0.41s]
[2025-10-03T16:17:29.860+0000] {subprocess.py:93} INFO - [0m16:17:29
[2025-10-03T16:17:29.863+0000] {subprocess.py:93} INFO - [0m16:17:29  Finished running 2 data tests in 0 hours 0 minutes and 1.42 seconds (1.42s).
[2025-10-03T16:17:29.967+0000] {subprocess.py:93} INFO - [0m16:17:29
[2025-10-03T16:17:29.969+0000] {subprocess.py:93} INFO - [0m16:17:29  [32mCompleted successfully[0m
[2025-10-03T16:17:29.970+0000] {subprocess.py:93} INFO - [0m16:17:29
[2025-10-03T16:17:29.972+0000] {subprocess.py:93} INFO - [0m16:17:29  Done. PASS=2 WARN=0 ERROR=0 SKIP=0 NO-OP=0 TOTAL=2
[2025-10-03T16:17:31.136+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-10-03T16:17:31.139+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-03T16:17:31.188+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dbt_dag, task_id=dbt_tests, run_id=scheduled__2025-10-02T00:00:00+00:00, execution_date=20251002T000000, start_date=20251003T161718, end_date=20251003T161731
[2025-10-03T16:17:31.258+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-10-03T16:17:31.281+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-03T16:17:31.284+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
